### DISCUSSION TITLE
Get date.future value in ISO 8601 (yyyy-MM-dd'T'HH:mm:ss.SSSxxx) format

### DISCUSSION THREAD

**Discussion by Biplab-Dutta** [mockoon/mockoon#1422] [Help] [answered]
I am trying to write a response template which required me to have a DateTime value in ISO 8601 format and the time needs to be ahead of the current time.
I tried `{{faker 'date.future'}}` but it gives the output as `Mon Feb 10 2025 03:47:21 GMT+0545 (Nepal Time)`. However, I'd like the output to be something like `2024-05-13T16:46:28.987386+05:45` which is a ISO 8601 format.
How can I do it?
Thanks.

**Comment by Biplab-Dutta** [comment]
I came across `{{dateTimeShift date=(now) format='yyyy-MM-dd'T'HH:mm:ss.SSSxxx' days=1}}`.
However I am not being able to apply escape character for quotes surrounding T. How can I do it?

**Reply by 255kb** [reply, marked as answer]
You can use the backslash: `{{dateTimeShift date=(now) format='yyyy-MM-dd\'T\'HH:mm:ss.SSSxxx' days=1}}`

**Reply by Biplab-Dutta** [reply]
Nope. I tried that. But it gave an error. Seemed like backslash isn't allowed in the format section.
I don't remember the error message but it was something like invalid escape character.

**Reply by Biplab-Dutta** [reply]
I tried again by adding backslash as you mentioned. It seems to be working now. I don't know why I was getting an error previously. Anyway marking it as answered.
